tests/Entity/BlockInteractorTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\PageBundle\Tests\Entity;
+
+use Doctrine\Persistence\ManagerRegistry;
+use PHPUnit\Framework\TestCase;
+use Sonata\PageBundle\Entity\BlockInteractor;
+use Sonata\PageBundle\Model\Block;
+use Sonata\PageBundle\Model\BlockManagerInterface;
+use Sonata\PageBundle\Model\Page;
+use Sonata\PageBundle\Model\PageInterface;
+
+class BlockInteractorTest extends TestCase
+{
+    /**
+     * @testdox It is returning a block list.
+     */
+    public function testLoadPageBlocks(): void
+    {
+        //Mock
+        $managerRegistryMock = $this->createMock(ManagerRegistry::class);
+        $blockManagerInterfaceMock = $this->createMock(BlockManagerInterface::class);
+        //TODO it's not possible to pass the "PageInterface", because this method "disableBlockLazyLoading" does not exist there.
```

Comment:
Prefer a NEXT_MAJOR comment, change to PageInterface.

Reply:
Yeah I did it 😄